content/blog/tidymodels-2022-survey/index.Rmd

```diff
@@ -0,0 +1,51 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2022-survey
+title: tidymodels 2022 developer survey
+date: 2021-09-27
+author: Max Kuhn
+description: >
+    We are conducting our second tidymodels devleoper survey. Please give us your
+    feedback!
+
+photo:
+  url: https://unsplash.com/photos/ChUHmPPTnLQ
+  author: Djim Loic
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [other] 
+tags: [survey,tidymodels]
+---
+
+In 2020, we created a developer survey to get community impact on how we prioritize our projects. [The results](https://connect.rstudioservices.com/tidymodels-priorities-survey/README.html) gave us a good sense of which items people were most interested in. The top three requests were:
+
+* Model stacking! This ended up resulting in the [award](https://twitter.com/simonpcouch/status/1347288263472553984) winning [stacks package](https://stacks.tidymodels.org/) by our intern Simon Couch. 
+* Model Operations (deploying and monitoring models) was important. We _might_ be working on this from [multiple fronts](https://twitter.com/juliasilge/status/1440784933576851456?s=20). 
+* Adaptive resampling and better parallel processing. The [finetune package](https://finetune.tidymodels.org/) solved the first part and some work [on the tune package](https://github.com/tidymodels/tune/pull/305) resolved the second.  
+
+Almost everything on the list has been finalized or is currently in-progress. 
+
+Most of our time right now is spent on model operations, survival analysis, and case weights. That work will proceed through the end of the year (at least). It's time to get more feedback on new features for next year. 
```

Comment:
```suggestion
Most of our time right now is spent on model operations, survival analysis, and case weights. That work will take us through the end of the year (at least). Since this survey seemed to work well last year, we've decided to use it again to prioritize features for next year.
```